src/authenticationservices.cs

```diff
@@ -252,16 +309,16 @@ interface ASPasswordCredentialIdentity : NSCopying, NSSecureCoding {
 		ASPasswordCredentialIdentity Create (ASCredentialServiceIdentifier serviceIdentifier, string user, [NullAllowed] string recordIdentifier);
 
 		[Export ("serviceIdentifier", ArgumentSemantic.Strong)]
-		ASCredentialServiceIdentifier ServiceIdentifier { get; }
+		new ASCredentialServiceIdentifier ServiceIdentifier { get; }
```

Comment:
What is the context of this 'new'? 
I know about the 'new' keyword to hide something from the base class but is that the context here?

Reply:
It does look like this is the same context: https://learn.microsoft.com/en-us/xamarin/cross-platform/macios/binding/binding-types-reference#newattribute.

Could I still get more clarification since the base type is NSObject?

Reply:
Because it is inheriting from ASCredentialIdentity, which already have some of this methods. This is just so that we remove a warning from the compilation of the intermediate assembly.